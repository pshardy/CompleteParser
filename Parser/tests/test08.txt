
VAR
  a, b, r;
  c : ARRAY[100];
{
   a = 10;
   b = 5;
   c[a] = 10;
   c[a + b] = 20;
   c[a * b] = a + b * (c[a] + c[a + b]);
   r = c[a];
   print r;
   r = c[a + b];
   print r;
   r = c[a*b];
   print r;
   r = c[10] + c[15] + c[50];
   print r;
}

