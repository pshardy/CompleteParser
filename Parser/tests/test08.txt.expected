10
20
160
190

